libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<section>
+  <div id="map-section">
+    <tamu-gisc-drawer [width]="'25rem'" [position]="'left'">
+      <div class="content">
+        <h3>Map Controls</h3>
+        <label class="switch">
+          <input type="checkbox" id="csToggle" (click)="stateToggle(this)">
+          <span class="slider round"></span>
+        </label>
+        Counties/State
+        <br>
+        <label class="switch">
+          <input type="checkbox" id="mortalToggle" (click)="mortalityToggle(this)">
+          <span class="slider round"></span>
+        </label>
+        Infection Rate/Mortality Rate
+      </div>
+    </tamu-gisc-drawer>
+    <tamu-gisc-mapbox-map
+      token="pk.eyJ1Ijoid2hhdHppdG9veWEiLCJhIjoiY2poNzI3M2M0MDNnbjMzcWVtYmZnNXExNiJ9.K-nM-JaZnZvAfutkLQitbg"
+      [center]="['-98', '38.88']">
+    </tamu-gisc-mapbox-map>
+    <div class = 'right-sidebar'>
+      <h3>COVID-19 Data</h3>
+      <div id= "info-box">
+        <p>Hover over the map to see info</p>
+      </div>
+      <div id="cases-legend" class="legend" style="display: block;">
```

Comment:
No inline styles.

Reply:
gave these id and assigned them a color in the css file